### ISSUE TITLE
Clothes - Formal Dress

### ISSUE THREAD

**Issue by Chrissenpai** [Monika-After-Story/MonikaModDev#3863]
Behold, my first commission, here is when Monika has prepared herself to dance with you, coming soon with all the poses
![image](https://user-images.githubusercontent.com/40907045/53519543-8494ca00-3aa1-11e9-828c-42c5c6dd2fbe.png)

#3174

**Comment by Shiro2564** [comment]
it looks wonderfull

**Comment by antimockingjay** [comment]
Oh my god, she looks amazing. I love the color, too!! Since we can't see the shoulders with her arms there, I have to ask: Does the dress have straps, or is it entirely off-the-shoulder? Either way, it's gorgeous! I'm also curious about the length of the dress. It's formal, so I'm guessing full-length?

I'd love to see this with the ribbon color variants! Does the color match any of the ribbons? I hardly see any of them so I wouldn't know. If it doesn't, maybe we need a ribbon that will match the dress! It could be such a cute combo!

**Comment by AsuraShun** [comment]
Love this to bits

**Comment by Chrissenpai** [comment]
@antimockingjay I still don´t know yet how to visualize the ribbon color, and is entirely off the shoulder, I have drawn some of the arms down pose and it fits

**Comment by jmwall24** [comment]
the sleeves could use some work, especially the parts by her hands and wrist. the texture is very flat there. ends up looking like just a recolor of her arm there.  it might just be how the sleeves ends.

**Comment by MaximumRustler** [comment]
She looks beautiful in it, love it so far.

**Comment by Chrissenpai** [comment]
@jmwall24 you´re right, the arms look flat compared to the rest of them, polishing that.

**Comment by Chrissenpai** [comment]
actualization of dress @jmwall24 
![image](https://user-images.githubusercontent.com/40907045/53532893-b1a7a380-3ac6-11e9-85c9-fe676680f856.png)

**Comment by jmwall24** [comment]
that helps a lot 👍

**Comment by Chrissenpai** [comment]
![image](https://user-images.githubusercontent.com/40907045/53534937-d3a52400-3ace-11e9-9940-9da9c491fa7f.png)

**Comment by EntonyEscX** [comment]
So beautiful!!!

**Comment by antimockingjay** [comment]
I might suggest removing the... I don't know what to call it. The curve, I guess? On the inside of the palm there. Basically what I'm saying is, fabric doesn't inherently stick to the skin. The sleeve seems to be pretty tight, so it wouldn't conform to the curve of her palm. Find a shirt or some other fabric and stretch it over anything that's concave, you'll see what I mean. It doesn't conform perfectly to the shape of what it's around. In the picture, though, it seems that it's sticking to the palm of her hand perfectly. The only time fabric really does that is if it's wet or something.

It's fine if you choose not to! This isn't, like, a demand of some kind. It's just a suggestion, based on what I've seen of fabric. If you wanna keep it as is, that's fine, it still looks absolutely amazing!! You've done a great job!!! ❤️

**Comment by Chrissenpai** [comment]
@antimockingjay 
![image](https://user-images.githubusercontent.com/40907045/53541358-7c5f7d80-3ae7-11e9-82a0-4b6266e743a8.png)

**Comment by softombre** [comment]
This looks amazing,,,

**Comment by bittercaramel** [comment]
woah its very good <3 
i would wear this outfit myself but monika looks great in it!  ~~kinda~~

**Comment by andersonantunes26** [comment]
this clothes could be added when we take the monika somewhere

![](https://user-images.githubusercontent.com/40907045/53541358-7c5f7d80-3ae7-11e9-82a0-4b6266e743a8.png)

**Comment by antimockingjay** [comment]
That's exactly what I meant! Also, in the latest two, is it just me or is there, like, some extra material at the sides? It almost looks like another layer had the dress with Monika's arms down, but without the color. Maybe I'm mistaken though?

**Comment by Orcaramelo** [comment]
This looks so cute!

**Comment by VirginiaRocks** [comment]
It's so beautiful!

**Comment by AsuraShun** [comment]
Beyond words right?

**Comment by LordBaaa** [comment]
Could I get the psd's of the your current progress pls?
